### DISCUSSION TITLE
Processo seletivo - Reprodução de PSD in loco.

### DISCUSSION THREAD

**Discussion by igorthurow** [frontendbr/forum#1016] [Discussão]
Tudo bem? Bom dia!

Pessoal, seguindo as dicas da issue [#968](https://github.com/frontendbr/forum/issues/968), passei pra última etapa de um processo seletivo de uma empresa de plataforma de e-commerce de POA, para função de dev Front.

O primeiro contato foi por telefone, o segundo eu tive que reproduzir um PSD, que pode ser visto [aqui ](http://igorthurow.me/MoovinChallenge/layout.png) na forma de PNG. 
Detalhe: Não podia usar nenhum framework de Front, portanto usei CSS Grid. E o resultado do meu trabalho [aqui](http://www.igorthurow.me/MoovinChallenge/).

Ontem, no e-mail que recebi do RH, não recebi maiores informações da terceira etapa do processo, a menina me disse apenas que presencialmente haverá um teste parecido com o home office, com média de 3h de duração. Ou seja, informação vaga.

Quero me preparar... Não sei se refaço minha tela para fins de estudo.. (pois foi dito que o teste in loco seria parecido), ou se estudo questões técnicas de JS, HTML e CSS.. Se estudo lógica? Vai ser a minha primeira entrevista onde avancei etapas no processo seletivo, não quero deixar a oportunidade passar. 

Vocês tem dicas? Vou estudar o dia e noite inteiras hoje.

**Comment by caioincau** [comment]
Acho mto bizarro o cara te mandar um PSD sem nem perguntar se você tem Photoshop, eu não tenho por exemplo.

Quanto a se preparar, eles vão querer ver você codar o que sabe na frente deles, dependendo de commo conduzirem pode ser legal, ou pode ser um parto por botarem pressão.

Minha dica é, no lugar de ficar varando de estudar, relaxe, descanse, se alimente bem, pois o que você tinha que aprender até o teste, já aprendeu.

**Comment by igorthurow** [comment]
@caioincau Acho que só aí ele já começa a aplicação de filtros para selecionar o candidato que esteja mais familiarizado com as ferramentas utilizadas pela empresa?!

**Comment by marcusviniciusf** [comment]
Concordo com o Caio. Você ja demonstrou seus conhecimentos no teste. E pelo que vi, não tem muito de JS, então acho que o básico já basta. Fica tranquilo, o seu teste está ótimo. E, se mesmo assim, não conseguir ficar quieto, pegue o teste e reflita, veja pontos que podem melhorar, anote-os, teste mais, ache bugs, corrija-os, e leve-os para os entrevistadores, porque certamente eles vão comentar algo do tipo

**Comment by igorthurow** [comment]
@MarcwL22 Ótima ideia! Anotar os pontos do meu teste.. como cada coisa foi feita, e nisso já ver se há algo que eu posso melhorar. Me sinto na faculdade na véspera da prova do semestre hahaha

**Comment by bernardodiasc** [comment]
> Minha dica é, no lugar de ficar varando de estudar, relaxe, descanse, se alimente bem, pois o que você tinha que aprender até o teste, já aprendeu.

Concordo, chega la na empresa de bouas, causa uma boa impressão, o resto vc ja tirou de letra :)

**Comment by felipefialho** [comment]
Fala @igorthurow, primeiramente, boa sorte :)

A galera já deu as principais dicas, dificilmente esse teste no local vai ser muito diferente do que você fez em casa. Talvez queiram observar melhor as técnicas que você curte usar ou como pesquisa quando se depara com problemas.

Não acho que seja o momento de estudar, pode encher sua cabeça de coisa e te deixar ansioso. Só descansa e faz o que você sabe fazer, o correto é a empresa saber avaliar sua postura para lidar com problemas e não só o resultado, é tão importante tanto.

Depois nos avise se conseguiu ;)

**Comment by igorthurow** [comment]
Muito obrigado senhores!! Como é bom ter esse apoio da comunidade. Pensa em ansioso haha.
Vou ficar tranquilo, dormir cedo, sossegar. Amanhã ser eu mesmo e mostrar o que posso fazer. Valeu MUITO. 
@LFeh Com certeza vou voltar e avisar vocês que consegui meu primeiro emprego hahaha. Obrigado velho.

**Comment by willycamargo** [comment]
Primeiramente parabéns pelo que já fez, está muito bom!

Eu, particularmente, não acho que deveriam aplicar mais um teste de 3 horas para provar que você é realmente bom. Acredito que o ideal seria eles abrirem seu código na sua frente e apontarem problemas e fazerem perguntas. Mas se é processo da empresa, fazer o que rs...

Os colegas acima já falaram tudo, apenas relaxe e vá para lá como se estivesse indo para a padaria comprar pão. Chegue lá e trate o recrutador como se fosse um amigo.

A tranquilidade e confiança serão muito bom nessa hora. Se esforce para mostrar o que sabe e não tenha vergonha de dizer que não sabe algo.

Caso te passem algo que você não sabe fazer, diga que você não consegue fazer de cabeça mas que com alguns minutos de pesquisa provavelmente chegará a uma solução legal para o problema. Afinal, os melhores desevolvedores que eu conheço são os que sabem pesquisar de forma correta e rápida 😄 

Boa sorte, parceiro! 👏

**Comment by igorthurow** [comment]
Pessoal!
Demorei pra atualizar aqui como forma de conter as expectativas.
Bom, fiz a entrevista neste dia, quinta feira da semana passada. E senti que me saí bem, inclusive não precisei realizar o teste técnico, o teste home office valeu como teste final.
Passou-se uma semana, e na quinta passada o RH retomou o contato, e na sexta me ligaram informando que começo dia 02/05!!!
Estou muito feliz e gratificado! Muito obrigado a todos, a comunidade me ajudou muito nessa caminhada.
(Estou repostando esta mensagem, havia apagado com receio de ter algum imprevisto na contratação, mas hoje confirmei que deu tudo certo!)

**Comment by felipefialho** [comment]
Parabéns @igorthurow! 🤘

Sucesso na nova empresa.

**Comment by igorthurow** [comment]
Cara! Muito obrigado @LFeh 
Sinceramente este fórum foi um divisor de águas na minha carreira! Agora só vai!

**Comment by carloshenriqueribeiro** [comment]
Parabéns man!

Carlos Ribeiro

**Comment by caioincau** [comment]
Sucesso man, parabéns!

**Comment by igorthurow** [comment]
@carloshenriqueribeiro  @caioincau  Muito obrigado senhores. Primeiro "Desenvolvedor Front-End" na CTPS não é todo dia hahahaha. Não da pra expressar o sentimento de conquista que eu tô agora.

**Comment by felipefialho** [comment]
@igorthurow https://twitter.com/LFeh/status/986591097752350721
